apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/TraceSnippets.cs

```diff
@@ -175,6 +197,53 @@ public void Configure(IApplicationBuilder app)
         // End sample
     }
 
+#if NETCOREAPP2_0
+    internal class TraceClientFactoryTestApplication
+    {
+        private static readonly string ProjectId = TestEnvironment.GetTestProjectId();
+
+        // Sample: ConfigureHttpClient
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // The line below is needed for trace ids to be added to logs.
+            services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
+
+            // Replace ProjectId with your Google Cloud Project ID.
+            services.AddGoogleTrace(options =>
+            {
+                options.ProjectId = ProjectId;
+            });
+
+            // Register an HttpClient for outgoing requests.
+            services.AddHttpClient("weahter", c =>
```

Comment:
weahter => weather

But I wonder whether it would be worth making the sample not configure the client at all apart from the AddHttpMessageHandler call?

Reply:
Yep, the snippet it's clearer if we only configure the `AddHttpMessageHandler` call, focus would be on our particular bit of configuration.
Changing.

Reply:
Done with a but:
`services.AddHttpClient()` returns `IServiceCollection`, all the other overloads for `AddHttpClient()` return an `IHttpClientBuilder` which is what we need to be able to add our custom message handler.
So I used the minimum extra configuration, which is just naming the HttpClient. 
```
services.AddHttpClient("tracesOutgoing")
.AddHttpMessageHandler<UnchainedTraceHeaderPropagatingHandler>();
```
This makes sense in a way, if we want to have HttpClients with extra configuration we at least have to name them.